### PR TITLE
Add backup node

### PR DESCRIPTION
**Pull request by IvayloPapazov**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -220,6 +220,34 @@ module "aws_deploy-main_mon-eu-north-1" {
   }
 }
 
+module "aws_deploy-main_backup-eu-central-1" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.3.1"
+  env               = "main_backup"
```
Comment (dincho) [inline main.tf]: env = main
kind = backup

File: main.tf [hunk 2]
```diff
@@ -220,6 +220,34 @@ module "aws_deploy-main_mon-eu-north-1" {
   }
 }
 
+module "aws_deploy-main_backup-eu-central-1" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.3.1"
```
Comment (dincho) [inline main.tf]: Once https://github.com/aeternity/terraform-aws-aenode-deploy/commit/542cf48efabf646321186aa059dbc7692e0470ca is release in v2.4, bump the module version

File: aws.tf [hunk 3]
```diff
@@ -30,3 +30,9 @@ provider "aws" {
   region  = "eu-north-1"
   alias   = "eu-north-1"
 }
+
+provider "aws" {
```
Comment (dincho) [inline aws.tf]: Please reuse one of the above regions

**Review by dincho — CHANGES_REQUESTED** [review]

File: aws.tf (hunk 3 above)
Comment (IvayloPapazov) [reply aws.tf]: 7b0b985

File: main.tf (hunk 2 above)
Comment (IvayloPapazov) [reply main.tf]: 7b0b985

File: main.tf (hunk 1 above)
Comment (IvayloPapazov) [reply main.tf]: 7b0b985

**Review by dincho — APPROVED** [review]

**Review by dincho — APPROVED** [review]

**Review by venimus — APPROVED** [review]

**Pull request merged**